doc/cug.tex

```diff
@@ -1400,12 +1400,62 @@ \subsection{Remote Tasks}
 ' "/opt/cylc/bin/cylc" '"'"'job-submit'"'"\
 ' --remote-mode "/home/oliverh/cylc-run/tut.oneoff.remote/log/job/1/hello/01/job"'
 \end{lstlisting}
-(Don't be intimated by this - it really quite straightforward and would appear
-much simpler if the job log path was shorter!). Remote task job logs are
+(Don't be intimated by this - it is really quite straightforward and would
+appear much simpler if the job log path was shorter!). Remote task job logs are
 saved to the suite run directory on the task host, not on the suite host,
-although they can be retrieved by right-clicking on the task in the GUI. Rose
-(section~\ref{Rose}) provides a task event handler to pull logs back to
-the suite host.
+although they can be retrieved by right-clicking on the task in the GUI. If you
+want the job logs pulled back to the suite host automatically, you can set
+\lstinline@[[[remote]]]retrieve job log=True@:
```

Comment:
This looks funny when process because it is directly above a long-format code block.  Maybe expand it more in text, "you can set 'retrieve job log = True' under the task [[[remote]]] section:"?